docs/terminology.md

```diff
@@ -0,0 +1,22 @@
+| Variables          | A State field that holds experimental variables, which are defined according to name, type, units, allowed values, and range.               | Experimentalists, Experiment Runners, Theorists |
+| VariableCollection | Immutable metadata about dependent variables, independent variables, and covariates.                                                        | Experimentalists, Experiment Runners, Theorists |
+| Conditions         | A State field that defines what observations should be collected according to a specific combination of values of the independent variables | Experimentalists, Experiment Runners, Theorists |
+| Experiment Data    | A State field that holds specified conditions as well as the corresponding observations.                                                    | Experiment Runners, Theorists                   |
+| Model              | A State field that holds the the collection of best fit equations produced by theorists.                                                    | Theorists, Experimentalists                     |
+| Components         | The objects that can act on the State (e.g., experimentalists, experiment runners, theorists).                                              | Experimentalists, Experiment Runners, Theorists |
+| Experimentalist    | A component that outputs new conditions, which are intended to yield novel observations.                                                    | Experimentalists                                |
+| Theorist           | A component that takes in the full collection of conditions and observations and outputs models that link the two.                          | Theorists                                       |
+| Experiment Runner  | A component that takes in conditions and collects corresponding observations.                                                               | Experiment Runners                              |
+| Wrapper            | Special functions that make the components of AutoRA able to operate on State objects.                                                      | Experimentalists, Experiment Runners, Theorists |
+| Workflow           | A collection of tools that enable closed-loop empirical research with the AutoRA framework.                                                 | Experimentalists, Experiment Runners, Theorists |
+| Cycle              | A workflow tool that allows AutoRA components to be chained together in serial loops.                                                       | Experimentalists, Experiment Runners, Theorists |
+| Cylc               | A workflow engine for cycling systems that orchestrates distributed workflows of interdependent cycling tasks.                              | Experimentalists, Experiment Runners, Theorists |
```

Comment:
I'd consider dropping "Cylc" here – it's just one way to run a cycle, and I don't think it's the most important way. We touch on Cylc in the specific workflow documentation, and I think that's probably enough.

Reply:
sounds good to me! per [instruction](https://github.com/AutoResearch/autora/issues/516#issuecomment-1716300791) in the task, i was going off this [page](https://docs.google.com/presentation/d/1OdP4M2ByW8GKTFvcCgDG5HDzUwHKdFEhj1ddH4ClvEw/mobilepresent?slide=id.g27e19158dea_1_0)